homeassistant/components/knx/sensor.py

```diff
@@ -54,6 +54,16 @@ def device_class(self) -> str | None:
             return device_class
         return None
 
+    @property
+    def extra_state_attributes(self) -> dict[str, Any] | None:
+        """Return device specific state attributes."""
+        attr: dict[str, Any] = {}
+
+        if self._device.last_telegram is not None:
+            attr[ATTR_SOURCE] = str(self._device.last_telegram.source_address)
+            attr[ATTR_LAST_KNX_UPDATE] = str(self._device.last_telegram.timestamp)
```

Comment:
Do we need to covert the time to UTC here too?

Reply:
Yep, forgot that part sorry. Fixed now.